frontend/src/store/company/actions.ts

```diff
@@ -0,0 +1,41 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { actions } from './slice';
+import { ActionType } from './common';
+import { ICompany } from 'common/interfaces/company/company';
+import { company as companyApi } from 'services';
+
+const add_companyAsync = createAsyncThunk(
+  ActionType.ADD_COMPANY,
+  async (newCompany: ICompany, { dispatch }) => {
+    const result = await companyApi.addCompany(newCompany);
+
+    if (result) {
+      const { token, company } = result;
+
+      window.localStorage.setItem('companytoken', token);
```

Comment:
set token instead of existing `StorageKey.TOKEN`
https://github.com/BinaryStudioAcademy/bsa-winter-2022-growup/blob/develop/frontend/src/common/enums/app/storage-key.enum.ts#L1